torchvision/prototype/transforms/_auto_augment.py

```diff
@@ -222,15 +252,14 @@ def _get_policies(
                 (("Solarize", 0.7, 2), ("TranslateY", 0.6, 7)),
                 (("ShearY", 0.8, 4), ("Invert", 0.8, None)),
                 (("ShearX", 0.7, 9), ("TranslateY", 0.8, 3)),
-                (("ShearY", 0.8, 5), ("AutoContrast", 0.7, None)),
+                (("ShearY", 0.8, 5), ("Autocontrast", 0.7, None)),
                 (("ShearX", 0.7, 2), ("Invert", 0.1, None)),
             ]
         else:
             raise ValueError(f"The provided policy {policy} is not recognized.")
 
     def forward(self, *inputs: Any, params: Optional[Dict[str, Any]] = None) -> Any:
         sample = inputs if len(inputs) > 1 else inputs[0]
-        params = params or self._get_params(sample)
```

Comment:
Do we need the params input on the method at all?

Reply:
Nope. I already removed it for all other transforms, but forgot about it here.